spec/tools/environment_builders/openstack_environments.rb

```diff
@@ -23,6 +23,12 @@ def usage(s)
 
 OBFUSCATED_PASSWORD = "password_2WpEraURh"
 OBFUSCATED_IP = "1.2.3.4"
+OBFUSCATED_DATE = "Fri, 20 Nov 2015 08:24:54 GMT"
+OBFUSCATED_X_AUTH_TOKEN = "49ab69a55ee24d4283c8a229e4c11541"
+OBFUSCATED_REQUEST_ID = "req-8bcce180-cfff-433e-bae0-95ba29666425"
+OBFUSCATED_TRANS_ID = "txaac826f4cca84bfab931b-00564f33be"
+OBFUSCATED_TIMESTAMP = "1448025167.35155"
+OBFUSCATED_AUDIT_IDS = ["bobuzZI8S2e7H0w0Ydi0XQ"]
```

Comment:
Calling it obfuscation is confusing.  obfuscation means that you have secure values you are trying to hide by making it hard to read.  Maybe `CONSISTENT_` is better because you are trying to make things consistent.